content/en/real_user_monitoring/data_collected/user_action.md

```diff
@@ -16,16 +16,42 @@ further_reading:
   text: "Datadog Standard Attributes"
 ---
 
-A custom user actions is a custom event generated for a given user action. [Add one by instrumenting your code][1].
+## Automatic Collection of User Actions
+Real User Monitoring (RUM) SDKs automatically detect user interactions performed during a user journey. Set the `trackInteractions` [initialization parameter][1] to `true` to enable this feature.
```

Comment:
It could be nice to be consistent by using either `interaction` or `user action`